devito/operator/operator.py

```diff
@@ -939,21 +955,23 @@ def _emit_apply_profiling(self, args):
             if metrics:
                 perf("Global performance: [%s]" % ', '.join(metrics))
 
+            # Same as above, but excluding the setup phase, e.g. the CPU-GPU
```

Comment:
So the data transfer time from device to host is not included ?
is it reported somewhere else?

Reply:
wdym ? this is just an additional line -- a line that does not include the setup phase

Reply:
I will cross-check the logs again